src/shared/models/inscription.model.ts

```diff
@@ -107,6 +121,10 @@ export function whenInscriptionType<T>(
     return branches.image();
   }
 
+  if (mimeType.startsWith('image/svg') && branches.svg) {
+    return branches.svg();
```

Comment:
Just for my understanding, what does `branches` mean here in the context of the inscription type? 

Is that the recursive types?